app/views/flood_risk_engine/enrollments/steps/_correspondence_contact_email.html.erb

```diff
@@ -2,12 +2,12 @@
   <%= form_group_and_validation(form, :email_address) do -%>
     <%= f.label :email_address,  t('.label') , class: 'form-label' %>
     <span class="form-hint">We’ll send the registration confirmation to this address</span>
```

Comment:
I think "We’ll send the registration confirmation to this address" needs to be pulled from the locale file rather than being entered directly.